generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -3418,6 +3418,15 @@ def write_to_teds_from_file(
                 physical_channel=physical_channel, file_path=file_path,
                 basic_teds_options_raw=basic_teds_options))
 
+
+    def hash_task_sequence(self, task_sequence):
+        if isinstance(task_sequence, tuple):
+            task_sequence = list(task_sequence)
+            task_sequence[0] = task_sequence[0].name
+            task_sequence = tuple(task_sequence)
+            return hash(task_sequence)
+        return hash(task_sequence.name)
```

Comment:
The interpreter should only be responsible for hashing task handles, not sequences.
```suggestion
    def hash_task_handle(self, task_handle):
        return hash(task_handle.name)
```